lib/pinpoint_supported_countries.rb

```diff
@@ -159,6 +159,10 @@ def trim_spaces(str)
     str.gsub(/\s{2,}/, ' ').gsub(/\s+$/, '')
   end
 
+  def trim_spaces_digits(str)
```

Comment:
Add a comment explaining why this is here?
```suggestion
  # Remove trailing footnote numbers from country names
  def trim_spaces_digits(str)
```